tests/python/test_repro.py

```diff
@@ -178,6 +178,7 @@ def fusion_func(fd: FusionDefinition) -> None:
             ),
         ]
         outputs = fd.execute(inputs)
+        fd.validate_with_auto_inferred_outputs(outputs, inputs)
```

Comment:
This two steps validation may be useful when user want to manually schedule and run a fusion, then validate results.